code/src/reducers/tasks.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+import moment from 'moment';
+import uniqid from 'uniqid'
+
+const tasks = createSlice({
+  name: 'todos',
```

Comment:
I would choose either task or todo in naming. Having both to describe the same thing made some parts of your code in the components hard to track back. No big flaw as it doesn't make any different to the user experience. Just something that can help readability for someone else like a colleague or peer programmer of some sort, using or working with your code.